redaxo/src/addons/phpmailer/pages/config.php

```diff
@@ -318,7 +326,11 @@
 $n = [];
 $n['label'] = '<label for="phpmailer-log">' . $addon->i18n('archive') . '</label>';
 $n['field'] = $sel_archive->get();
-$n['note'] = rex_i18n::rawMsg('phpmailer_archive_info', rex_mailer::logFolder(), '...'.substr(rex_mailer::logFolder(), -30));
+$n['note'] = rex_i18n::rawMsg('phpmailer_archive_info', rex_mailer::logFolder(), '…'.substr(rex_mailer::logFolder(), -30));
```

Comment:
hast du hier bewusst aus dem `...` ein ellipsis gemacht?

```suggestion
$n['note'] = rex_i18n::rawMsg('phpmailer_archive_info', rex_mailer::logFolder(), '...'.substr(rex_mailer::logFolder(), -30));
```

Reply:
ja

Reply:
So brechen sie nicht einzeln um auf leinen Screens und bleiben zusammen

Reply:
Alternativ könnte man wie in https://github.com/redaxo/redaxo/pull/3937 den vollen Pfad listen